tests/Doctrine/Tests/Common/Reflection/ExampleAnnotationClass.php

```diff
@@ -11,4 +11,11 @@
  */
 class ExampleAnnotationClass {
 
+  const foo = \stdClass::class;
+
+  /**
+   * Example with comment.
+   */
+  const bar = \stdClass::class;
```

Comment:
I've added a param to the data provider for the existing annotation test and added a new class for the new test case.